source/parser.hera

```diff
@@ -5047,8 +5054,20 @@ Init
 # EOS/Nested rules are used carefully and if we only compare to the
 # pushed value.
 Indent
-  ( "  " / "\t" )* ->
-    const level = $1.length
+  ( " " / "\t" )* ->
```

Comment:
Here you can use `[ \t]*` for the rule. This will allow the parser to spend more time in optimized regex land. Then in the handler `$0` will be the entire string match so `$0.split('').forEach` or similar.

Reply:
Done.